source/assets/stylesheets/docs/_base.sass

```diff
@@ -41,9 +41,6 @@ h1,h2,h3,h4,h5,h6
 .index[class*="ls-theme-"] p a:not([class*="ls-btn"])
   color: #001e45
 
-ul, ol
```

Comment:
Is broken this page.
![ul](https://cloud.githubusercontent.com/assets/1235904/5877060/c01ccfd0-a303-11e4-8cba-98bfb5ea091d.jpg)